site/content/contribute/getting-started/code-reviews.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Code Reviews"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+All changes to the product must be reviewed.
+
+* User experience changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* Code changes must be reviewed by at least two [core committers](http://localhost:1313/contribute/getting-started/core-committers/#core-committers).
+* Documentation changes should be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers) and at least one [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+If you are not a core committer
+-------------------------------
+
+1. Submit your pull request.
+2. Reach out to the author of the ticket your PR resolves.
+    * Typically this means [mentioning](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#mentioning-people-and-teams) their username as a comment on the pull request.
+    * Product managers and core committers are often on the lookout for new PRs, and may handle this for you automatically.
+    * When in doubt, ask for help in the [Developers](https://community.mattermost.com/core/channels/developers) channel on our community server.
+3. Wait for a review.
+    * The ticket author, another core committer or a product manager will assign reviewers and label your pull request appropriately.
+    * Expect some interaction with at least one reviewer within 5 business days.
+    * Keep in mind that Mattermost Core Committers are geographically distributed around the world and likely in a different time zone than your own.
+4. Make any necessary changes.
+    * If a reviewer requests changes, your pull request will disappear from their queue of reviews.
+    * Once you've addressed the concerns, reach out with another mention or via our community server.
+5. Wait for your code to be merged.
+    * Once all reviewers have approved your changes, they will handle merging your code.
+
+If you are a core committer seeking a review
+--------------------------------------------
+
+1. Submit your pull request.
+2. Assign a [product manager](/contribute/getting-started/core-committers/#product-managers) to your review and label your pull request with `1: PM Review`.
+    * If your changes do not affect the user experience, you may skip this step.
+    * Wait for their review before continuing, to avoid churn if changes are requested.
+3. Assign two [core committers](/contribute/getting-started/core-committers/) to your review and label your pull request with `2: Dev Review`.
+    * When picking your first core committer, consider someone with domain expertise relative to your changes. Sometimes GitHub will recommend a recent editor of the code, but often you must rely on your own intuition from past interactions.
```

Comment:
Instead of relying on own intuition, would basing it off of the "Dev Areas" in Core Committer documentation be helpful? https://github.com/mattermost/mattermost-developer-documentation/blob/master/site/content/contribute/getting-started/core-committers.md

Reply:
Though at a closer review, the dev areas may not be up to date, and are less specific for more recent core committers.

Reply:
Yeah in reality most people are going to end up using their past experience to make the decision